code/Controllers/ContentController.php

```diff
@@ -195,11 +196,16 @@ public function handleRequest(HTTPRequest $request)
             if (class_exists('Translatable')) {
                 Translatable::disable_locale_filter();
             }
+
+            $filter = URLSegmentFilter::create();
```

Comment:
I think we should be consistent with what `SiteTree` does. In the one place where it checks if multibyte character are allowed, it reads it from the config. https://github.com/silverstripe/silverstripe-cms/blob/097d2211507576dc618782642017d5bce2312e9e/code/Model/SiteTree.php#L1942

If someone ends up using using the injector to define their own `URLSegmentFitler`, you could end up in a situation where `SiteTree` and `ContentController` behave differently.

Reply:
I've fixed that by correcting `SiteTree` to use `URLSegmentFilter::create()` instead. The instance is already used in a few other places where the logic is actually applied, this SiteTree line only adds some field descriptions.